Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -138,7 +138,7 @@
     <!-- XTXT: risk card - no contact yet -->
     <string name="risk_card_body_contact">"No exposure up to now"</string>
     <!-- XTXT: risk card - no exposures with low risk -->
-    <string name="risk_card_body_contact_low_risk">"No exposure with low risk so far"</string>
```

Comment:
If the string is removed in values.de it should be removed in all other languages (in my opinion), that means that line 140 and 141 have to be deleted.